### PR TITLE
Fix MERCURE_JWT_SECRET env var which was not set in the Travis CI build

### PR DESCRIPTION
**Pull request by teohhanhui**
Also add instructions for user on how to generate MERCURE_JWT_SECRET easily

"Unsecure" -> "Insecure"

### CODE REVIEW

File: ci/before_deploy [hunk 1]
```diff
@@ -17,8 +17,10 @@ fi
 # Generate random key & jwt for Mercure if not set
 if [[ -z $MERCURE_JWT_KEY ]]; then
     npm install --global "@clarketm/jwt-cli"
-    export MERCURE_JWT_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-    export MERCURE_JWT=$(jwt sign --noCopy '{"mercure": {"publish": ["*"]}}' $MERCURE_JWT_KEY)
+    MERCURE_JWT_KEY=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+    export MERCURE_JWT_KEY
```
Comment (teohhanhui) [inline ci/before_deploy]: https://github.com/koalaman/shellcheck/wiki/SC2155

File: ci/before_deploy [hunk 2]
```diff
@@ -17,8 +17,10 @@ fi
 # Generate random key & jwt for Mercure if not set
 if [[ -z $MERCURE_JWT_KEY ]]; then
     npm install --global "@clarketm/jwt-cli"
-    export MERCURE_JWT_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-    export MERCURE_JWT=$(jwt sign --noCopy '{"mercure": {"publish": ["*"]}}' $MERCURE_JWT_KEY)
+    MERCURE_JWT_KEY=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```
Comment (teohhanhui) [inline ci/before_deploy]: https://github.com/koalaman/shellcheck/wiki/SC2002

File: ci/before_deploy (hunk 1 above)
Comment (dunglas) [reply ci/before_deploy]: We should add shellcheck to this repo btw

File: docker-compose.yml [hunk 3]
```diff
@@ -67,7 +67,7 @@ services:
     image: dunglas/mercure
     environment:
       # You should definitely change all these values in production
-      - JWT_KEY=!UnsecureChangeMe!
+      - JWT_KEY=!InsecureChangeMe! # You have to change MERCURE_JWT_SECRET in api/.env when you change this. You can put the old value of MERCURE_JWT_SECRET into the debugger on https://jwt.io/ and put the new value of JWT_KEY in the "secret" field to obtain the new encoded value for MERCURE_JWT_SECRET
```
Comment (dunglas) [inline docker-compose.yml]: 👍 but we should also change this typo everywhere in dunglas/mercure for consistency.

**Pull request merged**